modules/launcher/src/main/java/org/mule/runtime/module/launcher/ToolingServiceAware.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.launcher;
+
+import org.mule.runtime.module.tooling.api.ToolingService;
+
+/**
+ * Enables {@link ToolingServiceAware} injection.
+ *
+ * @since 4.0
+ */
+public interface ToolingServiceAware
```

Comment:
It seems it was not clear by the whole review, but this is just to be consumed by remote clients and the set of services for those remote clients seems to make sense to be found under the same Service. Perhaps when more services appear we can review if we want to change it?. Mainly because we just have a couple of this services in the container and for tooling we would end up creating multiple.